src/Files.App/Actions/Global/EnterCompactOverlayAction.cs

```diff
@@ -0,0 +1,48 @@
+﻿using CommunityToolkit.Mvvm.ComponentModel;
+using CommunityToolkit.Mvvm.DependencyInjection;
+using Files.App.Commands;
+using Files.App.Contexts;
+using Files.App.Extensions;
+using Microsoft.UI.Windowing;
+using System.ComponentModel;
+using System.Threading.Tasks;
+using Windows.Graphics;
+using Windows.System;
+
+namespace Files.App.Actions
+{
+	internal class EnterCompactOverlayAction : ObservableObject, IAction
```

Comment:
@QuaintMako should we be grouping all the compact overlay actions together or should they be separate?

Reply:
I'd say separated, they are slightly different in their objectives.